Sources/MongoSwift/MongoError.swift

```diff
@@ -243,41 +243,40 @@ private func getBulkWriteErrorFromReply(
         withResult result: BulkWriteResult? = nil,
         withWriteConcernError wcErr: WriteConcernError? = nil) throws -> MongoError? {
     let decoder = BSONDecoder()
-    var insertedIds = result?.insertedIds
 
-    var bulkWriteErrors: [BulkWriteError]?
```

Comment:
When using fail points, sometimes empty (but non-nil) writeErrors and writeConcernErrors arrays are returned for parsing. The way this function was written assumed that wasn't possible and has been rewritten to accommodate those cases.